site/guide/generate-model-documentation.qmd

```diff
@@ -18,11 +18,11 @@ graph LR
 
 1. **Develop your model**: In your existing developer environment, build one or more candidate models that need to be validated. This step includes all the usual activities you already follow as a model developer. 
 
-2. **Generate model documentation**: With the ValidMind Developer Framework, generate automated model documentation and run validation tests. This step includes making use of the automation and testing functionality provided by the framework and uploading the output to the Platform UI. You can iteratively regenerate the documentation as you work though the next step of refining your documentation.
+2. **Generate model documentation**: With the {{< var vm_framework >}}, generate automated model documentation and run validation tests. This step includes making use of the automation and testing functionality provided by the framework and uploading the output to the {{< var vm_ui >}}. You can iteratively regenerate the documentation as you work though the next step of refining your documentation.
 
-3. **Refine model documentation**: In the ValidMind Platform UI, review the generated documentation and test output. Iterate over the documentation and test output to refine your model documentation. Collaborate with other developers and model validators to finalize the model documentation and get it ready for review. 
+3. **Refine model documentation**: In the {{< var vm_platform >}}, review the generated documentation and test output. Iterate over the documentation and test output to refine your model documentation. Collaborate with other developers and model validators to finalize the model documentation and get it ready for review. 
 
-4. **Submit for review**: In the ValidMind Platform UI, you submit the model documentation for review which moves the documentation workflow moves to the next phase where a model validator will review it.
+4. **Submit for review**: In the {{< var vm_platform >}}, you submit the model documentation for review which moves the documentation workflow moves to the next phase where a model validator will review it.
```

Comment:
> Some exceptions are references to these terms in alt text, Mermaid charts and things like internal comments

GLOBAL COMMENT, look at line 29 for example: There are some references to "Developer Framework" that have not been changed over to variables. Likely a remnant of early discussions about product naming when we (incorrectly) used that as a proper noun. 

Just a handful of these that need to be fixed: 

```
10 results - 3 files

site/guide/generate-model-documentation.qmd:
   26  
   27: Before you can use the Developer Framework, you need to verify that the current documentation template contains all the necessary tests for the model you are developing: 
   28  
   29: - The template might already be sufficient and you only need to run the template within the Developer Framework to populate documentation. 
   30  
   31: - Or, more likely, the template might need additional tests that you can add these tests via the Developer Framework. 
   32  

   50     
   51:    - For functionality provided by the Developer Framework: [Add the relevant tests or content blocks](work-with-content-blocks.qmd) for the model use case. 
   52      

   75  
   76: Next, go to **With the Developer Framework, Step 2**.  {{< fa arrow-right >}}
   77  
   78: ### With the Developer Framework
   79  

  104     
  105: 4.  Go to **With the Developer Framework, Step 3**. {{< fa arrow-right >}}
  106  
  107: 5. {{< fa arrow-right >}} After **With the Developer Framework, Step 6**, [add content blocks](work-with-content-blocks.qmd) to your model documentation: 
  108     

site/releases/2023-sep-27/highlights.qmd:
  143     * **Reworked the "Get Started" section** to include more conceptual information and an overview of the high-level workflows. [Try it ...](../../guide/get-started.qmd)
  144:    * **Revised the "Developer Framework"** section to provide an end-to-end overview of the workflow that model developers should follow as they adopt the framework.  [Try it ...](../../guide/get-started-developer-framework.qmd)
  145  

site/releases/2024-feb-14/highlights.qmd:
  113  
  114: We are now showing the name of the user who ran the action instead of a generic "Developer Framework" name whenever you generate documentation:
  115  
```

Reply:
Ah yes, I didn't catch all of the case-locked ones even though I did several passes.

Reply:
You resolved this conversation, but I don't see any commits that would address this?

Reply:
I thought we agreed that we would log another Story to address these additional changes on the huddle? 😅 If that's not OK I can try again!